src/components/AccountMenu/AccountMenu.tsx

```diff
@@ -0,0 +1,59 @@
+import { IconChevronDown, IconUser } from '@tabler/icons-react';
+
+import { Dropdown, DropdownContent, DropdownTrigger } from './../DropdownMenu';
+
+interface AccountMenuProps {
+  userName: string;
+  loggedAsRole: string;
+  logoutText: string;
+  children?: React.ReactNode;
+  className?: string;
+}
+
+export const AccountMenu: React.FC<AccountMenuProps> = ({
+  userName,
+  loggedAsRole,
+  children,
+  className = '',
+}) => {
+  return (
+    <div className={className}>
+      <Dropdown>
+        <DropdownTrigger
+          asChild
+          className="rounded-sm bg-surface-tertiary cursor-pointer
+            transition-shadow duration-150
```

Comment:
```suggestion
```
Transitions are currently not defined comprehensively, so let's not add it here.